src/gluonts/nursery/autogluon_tabular/predictor.py

```diff
@@ -130,6 +130,10 @@ def _to_forecast(
         start_timestamp: pd.Timestamp,
         item_id=None,
     ) -> Iterator[SampleForecast]:
+        if ag_output.size > self.prediction_length:
+            samples = ag_output.transpose()
+        else:
+            samples = ag_output.reshape((1, self.prediction_length))
         samples = ag_output.reshape((1, self.prediction_length))
         sample = SampleForecast(
```

Comment:
This is using `SampleForecast` to encode the prediction, which is OK for point prediction. But if the model explicitly outputs quantiles, it makes sense to use `QuantileForecast` instead. To do that, the predictor object will have to be configured with the quantile levels.

Reply:
yes. Let me explore `QuantileForecast`. Thanks for your suggestion!